src/tag/types.js

```diff
@@ -28,19 +29,23 @@ export type PropsT = {
   isFocused?: boolean,
   isHovered?: boolean,
   kind?: TagKindT,
+  variant?: TagVariantT,
   /** Component or String value for label of tag. Default is empty string. */
   children?: React$Node,
   /** The color theme to be applied to a Tag. Default is `KIND.primary`. */
   color?: string,
   /** Handler for events on Action button element. `children` provides which Tag was clicked. */
-  onActionClick: (
-    e: SyntheticEvent<HTMLInputElement>,
-    children?: React$Node,
-  ) => void,
+  onActionClick: (e: Event, children?: React$Node) => void,
+  /** Handler for events on Action button element. `children` provides which Tag was clicked. */
+  onActionKeyDown: (e: Event, children?: React$Node) => void,
+  /** Passing an onVlivk handler that also makes the tag clickable. */
```

Comment:
```suggestion
  /** Passing an onClick handler that also makes the tag clickable. */
```

Reply:
what use-case does `onActionKeyDown` solve? Seems to duplicate most of the functionality of `onActionClick`. Similarly with `onKeyDown`? I would personally hold off until someone needs it.

Reply:
Since we pass the keyDown handler to the action element (to handle keyboard navigation) we don't want to have it overwritten by users overrides, therefore, pulling it up to the top level